app/uk/gov/hmrc/customs/inventorylinking/export/controllers/CustomHeaderNames.scala

```diff
@@ -17,7 +17,8 @@
 package uk.gov.hmrc.customs.inventorylinking.export.controllers
 
 object CustomHeaderNames {
```

Comment:
just a thought shall we consider renaming this object ? Since it contains a value, `HeaderNames` might be misleading